src/Features/Core/Portable/InvertIf/AbstractInvertIfCodeRefactoringProvider.cs

```diff
@@ -570,6 +570,12 @@ private ImmutableArray<StatementRange> GetSubsequentStatementRanges(TIfStatement
                     var statementsAfterIf = statements.Skip(index + 1);
                     var ifBody = GetIfBody(ifNode);
 
+                    var currentParentClosingBrace = currentParent.ChildTokens().Last();
+                    var updatedLastStatement = statementsAfterIf.Last().WithTrailingTrivia(currentParentClosingBrace.LeadingTrivia);
+
+                    statementsAfterIf = statementsAfterIf.Take(statementsAfterIf.Count() - 1);
+                    statementsAfterIf = statementsAfterIf.Append(updatedLastStatement);
```

Comment:
doc what's going on here.  Specifically that you're trying to take any final structured trivia on the close brace, and moving it with the statements.